### ISSUE TITLE
Exporting 'root' note doesn't work

### ISSUE THREAD

**Issue by safesparrow** [zadam/trilium#1024]
Desktop version 0.42.2

Log file:
[trilium-2020-05-17.log](https://github.com/zadam/trilium/files/4640749/trilium-2020-05-17.log)

Steps:
- Select 'root' note, right-click, select 'export', choose destination file
Result:
- no file is created
- notification shows up with the following error:
```Export failed with following error: 'Cannot read property 'getNote' of null'. More details might be in the logs.```
- notification disappears automatically after a few seconds

Export works on any child of the root note.

Rather unfortunately, I was in the process of setting up a server instance.
I used the backup feature, didn't check the backup file exists, cleaned up trilium-data folder, set up server & sync and only realized there is no backup when I tried to import notes.

Rather unsafe backup procedure from me, I admit, but something that would help prevent this would be an error that is harder to miss - maybe a notification that doesn't go away until the user clicks 'ok', with some shade of red, would attract the attention of the user

**Comment by zadam** [comment]
Hello, thank you for reporting this issue, it has been fixed. Root is kind of special note (since it does not have parent) and sometimes there are root-specific bugs unfortunately.

> Rather unfortunately, I was in the process of setting up a server instance.  
> I used the backup feature, didn't check the backup file exists, cleaned up trilium-data folder, set up server & sync and only realized there is no backup when I tried to import notes.

I'm a bit confused about the process - why are you "cleaning up" the trilium-data folder?

To set up the sync, export & import should not be used (that would lead to duplication). There's a specific process described [here](https://github.com/zadam/trilium/wiki/Synchronization#how-to-set-up-synchronization) to set up sync correctly.

**Comment by safesparrow** [comment]
Thanks for the fix!

Sync was not working the last time I tried, most likely because of the cloudflare+nginx setup and broken sync wasn't leaving my desktop instance in a clear state. 
I was giving it a go with a differenet setup today. Thus I reckoned it would be easiest to start from scratch, with both desktop and server side empty, configure the account again, and then import those notes back into the desktop instance and let it sync with the server.
